src/server/api/controllers/users.controller.js

```diff
@@ -44,7 +44,12 @@ const getUserFavorites = async (user_id) => {
   try {
     const favorites = await knex('favorites')
       .join('products', 'products.id', 'product_id')
-      .select('products.*')
+      .select(
+        'products.id',
+        'products.name',
+        'products.picture',
+        'products.price',
```

Comment:
It looks ok. But can you add also `products.is_on_discount` and `products.discount_percent`, it will break some features that are related to favorite list.